<ui version="4.0" >
 <class>OutputInstrumentViewBase</class>
 <widget class="QDialog" name="OutputInstrumentViewBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Midi output key:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="midiOutputSpinBox" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>127</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="highProbabiltyCheckBox" >
         <property name="toolTip" >
          <string>If this instrument is to be used more often than others check this box.</string>
         </property>
         <property name="statusTip" >
          <string/>
         </property>
         <property name="text" >
          <string>Prefer always</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="preferWhenFastCheckBox" >
         <property name="toolTip" >
          <string>If this instrument is to be used more often when used often, check this checkbox.</string>
         </property>
         <property name="statusTip" >
          <string/>
         </property>
         <property name="text" >
          <string>Prefer when fast</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>  [low]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection" >
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text" >
          <string>  [high]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="lowRangeLabel" >
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Box</enum>
           </property>
           <property name="text" >
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="lowRangeSlider" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>127</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string> use in range </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="highRangeLabel" >
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Box</enum>
           </property>
           <property name="text" >
            <string>127</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="highRangeSlider" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>127</number>
           </property>
           <property name="value" >
            <number>127</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OutputInstrumentViewBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OutputInstrumentViewBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highRangeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>highRangeLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>215</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel" >
     <x>165</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowRangeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lowRangeLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel" >
     <x>25</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
